test/swar/BasicOperations.cpp

```diff
@@ -41,6 +41,15 @@ static_assert(
     multiplication_OverflowUnsafe_SpecificBitCount<3>(Micand, Mplier).value()
 );
 
+TEST_CASE("Jamie's totally working exponentiation :D") {
```

Comment:
Ah, thanks for the touch of humor, I think I will require good sense of humor in the style guide.  It cheers me to see the quality of the repository improving bit by bit.